docs/pages/versions/unversioned/sdk/app-auth.md

```diff
@@ -28,7 +28,7 @@ Below is a set of example functions that demonstrate how to use `expo-app-auth`
 
 <SnackInline>
 
-```tsx
+```js
```

Comment:
The code block is still broken. I think this should be `javascript` rather than `js`:

![Screenshot from 2020-02-25 16:00:06](https://user-images.githubusercontent.com/4579020/75253861-0cfa9780-57e8-11ea-80a2-cee11b859c46.png)